test/type/opaque_experimental.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-swift-frontend -enable-experimental-opaque-return-types -disable-availability-checking -typecheck -verify %s
```

Comment:
Nitpick: you can use %target-typecheck-verify-swift to expand to %target-swift-frontend -typecheck -verify %s